### DISCUSSION TITLE
How to export generated ts classes

### DISCUSSION THREAD

**Discussion by giacomo-cc** [wevm/wagmi#1861] [Q&A]
Hi team, I'm using wagmi for some time and the library is super developer friendly, congrats.

At the moment I'm facing a small issue that I can't easily solve. I have a mono repo with multiple projects, including:
- typescript sdk with json abi and typescript classes generated with `@wagmi/cli`
- next js app using the sdk

Currently I can't export the `generated.ts` file beacuse of a ts error:
_Return type of exported function has or is using name 'Overrides' from external module "my-path/node_modules/@wagmi/core/dist/contracts-b112c24e" but cannot be named. ts(4058)_

How can I solve this? Should I re-export wagmi from the sdk?

**Comment by tmm** [comment]
Hard to tell what is going on without looking at a minimal reproducible example, knowing what `wagmi`/`@wagmi/core` version you are using, etc.
